global.json

```diff
@@ -16,7 +16,6 @@
     "Microsoft.DotNet.Arcade.Sdk": "6.0.0-beta.21105.12",
     "Microsoft.DotNet.Helix.Sdk": "6.0.0-beta.21105.12",
     "Microsoft.DotNet.SharedFramework.Sdk": "6.0.0-beta.21105.12",
-    "Microsoft.FIX-85B6-MERGE-9C38-CONFLICT": "1.0.0",
     "Microsoft.NET.Sdk.IL": "6.0.0-preview.2.21115.2",
```

Comment:
You want to move that entry to the bottom so that the two auto-updated sections (arcade & runtime) are separated by hardcoded ones (Microsoft.Build.*).

Reply:
Oops - now I get what you meant. Fixed.